packages/reports/src/Containers/mobile-row-renderer.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { ProgressBar, ProgressSliderMobile, DataList, ContractCard, PositionsDrawerCard } from '@deriv/components';
+import {
+    isAccumulatorContract,
+    isMultiplierContract,
+    isVanillaContract,
+    isTurbosContract,
+    getContractDurationType,
+    getTimePercentage,
+    getCurrentTick,
+    getDurationPeriod,
+    getDurationUnitText,
+    getCardLabels,
+    toMoment,
+} from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+import { TColIndex } from 'Types';
+import moment from 'moment';
+
+type TRangeFloatZeroToOne = React.ComponentProps<typeof ProgressBar>['value'];
+type TPortfolioStore = ReturnType<typeof useStore>['portfolio'];
+type TDataList = React.ComponentProps<typeof DataList>;
+type TDataListCell = React.ComponentProps<typeof DataList.Cell>;
+type TUiStore = Pick<
+    ReturnType<typeof useStore>['ui'],
+    | 'addToast'
+    | 'current_focus'
+    | 'removeToast'
+    | 'setCurrentFocus'
+    | 'should_show_cancellation_warning'
+    | 'toggleCancellationWarning'
+>;
+
+export type TMobileRowRenderer = TUiStore & {
+    row?: TDataList['data_source'][0];
+    is_footer?: boolean;
+    columns_map: Record<TColIndex, TDataListCell['column']>;
+    getContractById: ReturnType<typeof useStore>['contract_trade']['getContractById'];
+    server_time: moment.Moment;
+    onClickCancel: (contract_id?: number) => void;
+    onClickRemove: TPortfolioStore['removePositionById'];
+    onClickSell: (contract_id?: number) => void;
+    measure?: () => void;
+};
+
+export const MobileRowRenderer = ({
+    row = {},
+    is_footer,
+    columns_map = {},
+    server_time = toMoment(),
+    onClickCancel,
+    onClickSell,
+    measure,
+    ...props
+}: TMobileRowRenderer) => {
+    React.useEffect(() => {
+        if (!is_footer) {
+            measure?.();
+        }
+    }, [row.contract_info?.underlying, measure, is_footer]);
+
+    if (is_footer) {
+        return (
+            <>
```

Comment:
We can omit `<>` here

Reply:
checked styles, doesn't seem to break without it, - removed it 👍🏻